.gitlab-ci.yml

```diff
@@ -67,7 +123,7 @@ Litmus-Chaos:
    - echo " Introducing-Chaos"
    - mkdir ~/.kube
    - cp  azure/config ~/.kube/config 
-   - kubectl create -f https://raw.githubusercontent.com/atulabhi/litmus/v0.7-RC1/apps/percona/chaos/openebs_volume_replica_failure/run_litmus_test.yaml
+   - kubectl create -f https://raw.githubusercontent.com/atulabhi/litmus/v0.7-RC1/apps/percona/chaos/openebs_volume_replica_failure/run_litmus_test.yml
    - chmod 755 ./script/chaos
```

Comment:
@dargasudarshan , let us use the same job template as others for chaos too - will help later when we modularize the gitlab yaml

Reply:
ok @ksatchit, I did not renamed the files as we need to rebase the atul's fork to have the latest changes. Once the repo is rebased I will make these changes